### ISSUE TITLE
Stm32 skipping packets in a similar projects

### ISSUE THREAD

**Issue by sx107** [slerpxcq/openuac2_fw#2]
Hi! I am making something similar on stm32h750 based on your project (I'll open source it once I'll be finished with it, the idea is to add several inputs/outputs), but I have issues with skipping packets even at 32bit / 48kHz at some USB ports that don't have anything high-bandwidth plugged. Did you encounter these issues as well? Did you modify anything in the stm32 usb library core and drivers?

**Comment by slerpxcq** [comment]
Hi, I have never implement this on H750. The core driver is not modified. I have heard the cache is causing some USB issues on H7 chips, maybe you can try to disable the cache?

**Comment by sx107** [comment]
Yes, I've already fixed that (the cache). Enabling USB DMA slightly made the situation better. Guess I'll dig into it some more. Thank you very much for your answer!

**Comment by sx107** [comment]
I will order a pcb with your project, but I've got to ask first: Did you have any similar problems? With STM32 straight up missing packets when they are sent in one SOF. Linux, for example, or "bad" usb ports seem to send 2-6 packets in one SOF and then silence. Did you observe similar behaviour? Or just that on some ports your project does not work?

**Comment by slerpxcq** [comment]
How can a USB port send 6 packet in one frame? As far as I know at most 3 packet could be sent per frame (when wMaxPacketSize[12..11] is set to 2). I have not experiment with more than 1 packet in one frame and have no idea whether STM32 support this. Anyway it is tested on both Linux and Windows at 384k/32bit and both are just working fine.
